pom.xml

```diff
@@ -269,6 +274,10 @@
   </dependencies>
 
   <repositories>
+    <repository>
+      <id>central</id>
+      <url>https://repo1.maven.org/maven2/</url>
+    </repository>
```

Comment:
Maven central is defined in the super pom. Could you check whether this is actually needed?